src/hooks/useScrollTop.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function UseScrollTop() {
```

Comment:
The name must be in camelCase like useScrollTop

Reply:
using in like this <UseScrollTop/> in main.tsx so can't rename it useScrollTop